src/dune_pkg/local_package.ml

```diff
@@ -21,6 +21,9 @@ type t =
   ; depopts : Package_dependency.t list
   ; pins : pins
   ; loc : Loc.t
+  ; build : OpamTypes.command list
+  ; install : OpamTypes.command list
+  ; dune_build : bool
```

Comment:
This makes for a somewhat inelegant type where if `dune_build` is set then `build` and `install` should be empty and cases where `dune_build` needs to be ignored. I assume if `dune_build` is true then `build` and `install` should be empty? What if they aren't?

I'd suggest something like:

```ocaml
type build_type =
  | Dune
  | Other of { build: OpamTypes.command list; install: OpamTypes.command list }
```

End then
```ocaml
; build_type : build_type
```